app/controllers/CheckingServiceController.scala

```diff
@@ -139,6 +141,9 @@ class CheckingServiceController @Inject()(authAction: AuthAction,
     } yield {
       Ok(check_csv_file(scheme, currentCsvFile.get.fileId)(request, messages, upscanResponse, appConfig, ersUtil))
     }) recover {
+      case e: NoSuchElementException =>
+        logger.warn((("[CheckingServiceController][showCheckCSVFilePage]: No files match status NotStarted")))
```

Comment:
this might be better as an error log unless it's already been logged at an error elsewhere